### PR TITLE
[Docs] Update ROADMAP.md (May/2024).

### PR DESCRIPTION
**Pull request by q82419**

### CODE REVIEW

**Comment by juntao** [comment]
Hello, I am a [code review bot](https://github.com/flows-network/github-pr-summary/) on [flows.network](https://flows.network/).

It could take a few minutes for me to analyze this PR. Relax, grab a cup of coffee and check back later. Thanks!

**Comment by alabulei1** [comment]
I  think we should add LLM-related features in the roadmap.

* Support Stable Diffusion
* Support burn.rs
* Support whisper.cpp
* Support Tensor RT
* Support Apple MLX
* Support Intel Intel Extension for Transformers
* Support llama.c
* Support running WasmEdge instance on  GPU

**Comment by alabulei1** [comment]
Hi @q82419 

Are there any updates on the ROADMA.md? We should also add what we have done since 2024 on the roadmap.md.

**Comment by q82419** [comment]
> Hi @q82419
> 
> Are there any updates on the ROADMA.md? We should also add what we have done since 2024 on the roadmap.md.

If the rule is okey, the above LLM related entries will be added in the next quarter.

@hydai  please take a look.

**Review by hydai — DISMISSED** [review]
LGTM

File: docs/ROADMAP.md [hunk 1]
```diff
@@ -2,39 +2,34 @@
...
+WasmEdge roadmap will be updated quarterly with the following steps.
 
-## How to add an item to the roadmap?
-
-Please open an issue to track any initiative on the roadmap of WasmEdge (Usually driven by new feature requests). We will work with and rely on our community to focus our efforts to improve WasmEdge.
+1. WasmEdge maintainers will raising an issue on the upcoming roadmap at least 14 days before the start of a new quarter.
+2. Any individual can request inclusion in the roadmap by commenting on the issue and volunteering as an assignee.
+3. WasmEdge maintainers will collect the proposed roadmap entries along with their respective timelines and assignees, then update them within the issue.
+4. Existing roadmap entries which not reach their deadlines will be retained in the new quarter.
+5. Roadmap entries that have reached their deadlines will be marked as completed or dropped, and the associated issues will be closed.
```
Comment (dannypsnl) [inline docs/ROADMAP.md:15]: Complete, drop, or update deadline? Since maybe the entry is delayed by accident.

Comment (hydai) [reply docs/ROADMAP.md:15]: We should figure out if there is any GitHub-Bot that can help to auto-close the stale proposals.

Comment (q82419) [reply docs/ROADMAP.md:15]: Sure, I think the deadline can be updated due to some reason. This can be discussed in the new quarter roadmap.

Comment (hydai) [reply docs/ROADMAP.md:15]: The timeslot is based on quarters now. How about:
1. Propose a quarter -> Stale
2. Propose two quarters -> Close
It should be fine to drop a proposal if we have no plan to implement it in half a year.

Comment (dannypsnl) [reply docs/ROADMAP.md]: Sounds reasonable.

Comment (q82419) [reply docs/ROADMAP.md]: Updated

**Review by hydai — APPROVED** [review]
LGTM. @alabulei1 PTAL.

**Comment by hydai** [comment]
https://github.com/ggerganov/llama.cpp/blob/master/.github/workflows/close-issue.yml

We can introduce something like this and set the period for the bot to do the stale and deprecated marks.

**Pull request merged**